pkg/machines/components/vnc.jsx

```diff
@@ -20,21 +20,43 @@ import React from 'react';
 import cockpit from 'cockpit';
 
 import { VncConsole } from '@patternfly/react-console';
+import { Dropdown, DropdownToggle, DropdownItem, DropdownSeparator } from '@patternfly/react-core';
 
 import { logDebug } from '../helpers.js';
+import { domainSendKey } from '../libvirt-dbus.js';
 
 const _ = cockpit.gettext;
+const Enum = {
+    KEY_BACKSPACE: 14,
+    KEY_F1: 59,
+    KEY_F2: 60,
+    KEY_F3: 61,
+    KEY_F4: 62,
+    KEY_F5: 63,
+    KEY_F6: 64,
+    KEY_F7: 65,
+    KEY_F8: 66,
+    KEY_F9: 67,
+    KEY_F10: 68,
+    KEY_F11: 87,
+    KEY_F12: 88,
+    KEY_RIGHTCTRL: 97,
+    KEY_RIGHTALT: 100,
+    KEY_DELETE: 111,
+};
```

Comment:
Can you put into comment where is this defined? Also does right/left ctrl/alt matter? (just I think left ones are usually default).

Reply:
It does not matter left/right I guess. I changed to left and added the link to the linux kernel source code where the keys are defined.